### PR TITLE
Loadingbg

### PR DESCRIPTION
**Pull request by LoadingBG**
<details>
<summary>Refactor <code>others.Embeds</code></summary>
Extract common logic from <code>#permissionError</code>, <code>#purgeError</code> and <code>#selfPunishError</code>
</details>
<details>
<summary>Use <code>implementation</code> instead of <code>compile</code> in <code>build.gradle</code></summary>
<code>compile</code> is <a href="https://docs.gradle.org/current/userguide/upgrading_version_5.html#changes_6.0">deprecated</a> since Gradle 6.0
</details>
<details>
<summary>Use spaces instead of tabulations for indentation</summary>
I use tabulations for indentation but seeing you've used spaces I changed the tabulations for consistency
</details>
<details>
<summary>Use <code>lombok.Data</code> annotation in <code>properties.command</code></summary>
<code>lombok.Data</code> annotation was used on <code>CommandConfig</code> and <code>CommandPrivilegesConfig</code> but not on the other classes. I made use of it for consistency
</details>

### CODE REVIEW

File: src/main/java/com/javadiscord/javabot/other/Embeds.java [hunk 1]
```diff
@@ -147,25 +90,20 @@ public static MessageEmbed configEmbed (Object ev, String title, String descript
...
 
         return eb.build();
     }
 
-    public static MessageEmbed configEmbed (Object ev, String title, String description, String imageLink, String value, boolean showValue, boolean channel) {
-
+    public static MessageEmbed configEmbed(Object ev, String title, String description, String imageLink, String value, boolean showValue, boolean channel) {
         return Embeds.configEmbed(ev, title, description, imageLink, value, showValue, channel, false);
     }
-
```
Comment (andrewlalis) [inline src/main/java/com/javadiscord/javabot/other/Embeds.java:159]: I think there should still be one line separating each method, so don't remove this one.

**Review by andrewlalis — CHANGES_REQUESTED** [review]
Thanks for the pull request! Just one comment which needs to be addressed before this is ready to merge. I also would like to mention that the whole embeds thing is meant to be somewhat deprecated as we move to use standardized simple responses via the `Responses` methods.

**Comment by LoadingBG** [comment]
`Embeds` is still used throughout the project in many places. I can try to replace `Embeds` with `Responses` and remove `Embeds` if you want.

File: src/main/java/com/javadiscord/javabot/other/Embeds.java (hunk 1 above)
Comment (LoadingBG) [reply src/main/java/com/javadiscord/javabot/other/Embeds.java:159]: sorry, that was a mistake, didn't see it

**Comment by andrewlalis** [comment]
> 
> 
> `Embeds` is still used throughout the project in many places. I can try to replace `Embeds` with `Responses` and remove `Embeds` if you want.

No, not yet at least. I need to make sure that we don't lose functionality first.

**Review by andrewlalis — APPROVED** [review]

**Pull request merged**